landoapi/repos.py

```diff
@@ -29,6 +29,11 @@
     ),
 )
 
+PRODUCT_DETAILS_LOCAL = "http://product-details.test"
+# TODO: update dev once json file is on merurcial server.
+PRODUCT_DETAILS_DEV = "https://product-details.mozilla.org"
+PRODUCT_DETAILS_PROD = "https://product-details.mozilla.org"
```

Comment:
after talking with zeid, decided to just hardcode the URLs per repo since it's just the base URL that's different and each repo will have a different URL regardless